Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -479,6 +479,36 @@
   categories:
   - UplinkUtility
 
+- type: listing
+  id: UplinkSyndicateEDaggerModule
+  name: uplink-syndicate-edagger-module-name
+  description: uplink-syndicate-edagger-module-desc
+  productEntity: BorgModuleEnergyDagger
+  cost:
+    Telecrystal: 4
+  categories:
+    - UplinkUtility
+
+- type: listing
+  id: UplinkSyndicateHyposprayModule
+  name: uplink-syndicate-hypospray-module-name
+  description: uplink-syndicate-hypospray-module-desc
+  productEntity: BorgModuleHypospray
+  cost:
+    Telecrystal: 5
```

Comment:
isnt this cheaper than a normal hypopen...

Reply:
I thought it should be cheaper because they can't hold a beaker of chemicals in their other hand all the time

Reply:
fair

Reply:
Buffed it by giving a bottle as well, but it’s only one tc above hypo because it has fewer options for follow up without sinking even more effort or tcs

Reply:
resolve this then its solved